src/basic/text.js

```diff
@@ -1,77 +0,0 @@
-/**
```

Comment:
I'm a bit curious. Did we have support for texts?

Reply:
We saw the other day that it didn't work, maybe it worked in some cases, maybe it was implemented but never tested.
I wouldn't hold this PR for the lack of text support since I think it was broken already and now it's just not implemented.

Reply:
Agree. I wanted to keep these comments here for future reference. Also, we will have this information in the `about_rendering.md` document.